step.yml

```diff
@@ -1,8 +1,47 @@
 title: Xcode build for simulator
 summary: Runs Xcode Build command for simulators.
 description: |
-  Runs Xcode Build command for simulators
-  for iOS, tvOS or watchOS.
+  This Step runs the `xcodebuild` command to build and deploy your iOS, tvOS, or watchOS app for a simulator. You can use this to perform quick tests for your app or to show it off in a browser to your clients. 
+  The Step generates the following outputs:
+  * `BITRISE_APP_DIR_PATH`: the path to the generated `.app` file.
+  * `BITRISE_APP_DIR_PATH_LIST`: the path to the generated `.app` file and the paths to every dependent target app. The paths are separated with the `|` character.
+  * `BITRISE_XCODE_BUILD_RAW_RESULT_TEXT_PATH`: the path to the log file of the raw build results.
+
+  The Step also creates an `.xctestrun` file which you can use to run tests.
+
+  Make sure to include this Step after the Steps that install the necessary dependencies, such as **Run Cocoapods Install** in your Workflow. 
+
+  ### Configuring the Step
+
+  To configure the Step:
```

Comment:
The output and this section seems to be a duplicate information. Aren't the input/output descriptions meant to give this guidance? I'm a bit afraid of duplicating input / output guidance as we'll need to keep track of both places when the step changes.